code/modules/surgery/organs/ears.dm

```diff
@@ -70,3 +70,24 @@
 	var/obj/item/organ/ears/ears = getorgan(/obj/item/organ/ears)
 	if(ears)
 		ears.minimumDeafTicks(value)
+
+
+/obj/item/organ/ears/cat
+	name = "cat ears"
+	icon = 'icons/obj/clothing/hats.dmi'
+	icon_state = "kitty"
+
+/obj/item/organ/ears/cat/adjustEarDamage(ddmg, ddeaf)
+	..(ddmg*2,ddeaf*2)
```

Comment:
Because lizard legs are optional, cat ears aren't.
(Yes you can choose not to be a catperson, but you can atleast -be- a lizard without having something like this)